modules/storage-persistent-storage-iscsi-enforcing-disk-quota.adoc

```diff
@@ -9,5 +9,5 @@ is one persistent volume. Kubernetes enforces unique names for persistent
 volumes.
 
 Enforcing quotas in this way allows the end user to request persistent
-storage by a specific amount (e.g, 10Gi) and be matched with a
+storage by a specific amount (for example, 10Gi) and be matched with a
```

Comment:
"10Gi" should be in back tics. Ex:

(for example, 10Gi) -> (for example, `10Gi`)